index.html

```diff
@@ -44,6 +44,217 @@ <h3>I'am a Software Developer</h3>
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+
+      <section class="card2">
+        <h3>Professional Art <br> Printing Data</h3>
+      
+        <p>A daily selection of privately 
+          personalized reads; no accounts or 
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+
+      <section class="card2">
+        <h3>Professional Art <br> Printing Data</h3>
+      
+        <p>A daily selection of privately 
+          personalized reads; no accounts or 
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+    </section>
+
+    <section class="container2">
+      <section class="aboutMe">
+        <h1>About me</h1>
+
+        <p>
+          Hello I’m a software developer! I can help 
+          you build a product , feature or website 
+          Look through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don’t 
+          hestiate to contact me.
+        </p>
+
+        <div>
+          <button class="orange-button orange-button3">Get My Resume</button>
+        </div>
+      </section>
+
+      <hr>
+
+      <section class="categories">
+        <div class="language">
+          <h3>Language</h3>
+
+          <ul class="list">
+            <li class="lang">Javascript</li>
+            <li class="lang">Ruby</li>
+            <li class="lang">Html</li>
+            <li class="lang">Css</li>
+          </ul>
+        </div>
+
+        <div class="frameworks">
+          <h3>Frameworks</h3>
+          
+          <ul class="list1">
+            <li class="frame">Bootstrap</li>
+            <li class="frame">Ruby on Rails</li>
+            <li class="frame">RSPec</li>
+            <li class="frame">CapyBara</li>
+            <li class="frame">Selenium</li>
+          </ul>
+        </div>
+

```

Comment:
- [ ]  Goodjob, I see that you have ordered the technologies using `ul` -`li`. The technology categories ("Languages", "Frameworks" and "Skills") is also a list and should use `ul`. 
_See my comment about semantic HTML above._

Reply:
Well addressed 💯